src/components/Home.tsx

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+
+export const Home:React.FC = () => {
+  return (
+    <div>
+      <h1 className="title">Home page</h1>
+    </div>
```

Comment:
You can use container div here

```suggestion
    <div classNames="container">
      <h1 className="title">Home page</h1>
    </div>
```